llvm/include/llvm/Cheerp/LinearMemoryHelper.h

```diff
@@ -130,6 +130,8 @@ class LinearMemoryHelper
 					return 'd';
 				case TypeKind::Float:
 					return 'f';
+				case TypeKind::Vector:
```

Comment:
can we assert that we are not in asmjs mode here? the information  should be available in this class

Reply:
The idea was that the break will cause the code to execute the unreachable. I'll add an assert if that makes the code cleaner.

Reply:
ah, I didn't notice that. then it's fine I think